src/test/java/edu/oakland/test/middleware01/MiddlewareGisInterfaceImplementationTests.java

```diff
@@ -0,0 +1,44 @@
+package edu.oakland.test.middleware01;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.Satellite;
+import edu.oakland.production.middleware01.MiddlewareGisInterface;
+import edu.oakland.production.middleware01.MiddlewareGisInterfaceImplementation;
+import edu.oakland.production.middleware01.MiddlewareGisProcessor;
+import edu.oakland.test.middleware01.MiddlewareGisProcessorStub;
+import java.lang.IllegalArgumentException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("MiddlewareGisInterfaceImplementation Unit Tests")
+public class MiddlewareGisInterfaceImplementationTests {
+
+  @Test
+  @DisplayName("Signal pass successful from interface to Display")
+  void passSignalStrengthUc1() {
+    Satellite sat = new Satellite("GPS0", 5);
```

Comment:
Generally we don't want to hard code values for testing, and instead want to generate random cases, but this if fine for what you're doing because it is a pass through.